html/elements/script.json

```diff
@@ -558,13 +558,16 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "61"
+                  "version_added": "61",
+                  "notes": "Module scripts without the <code>async</code> attribute do not load when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
                 },
                 "chrome_android": {
-                  "version_added": "61"
+                  "version_added": "61",
+                  "notes": "Module scripts without the <code>async</code> attribute do not load when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
                 },
                 "edge": {
-                  "version_added": "16"
+                  "version_added": "16",
+                  "notes": "Module scripts without the <code>async</code> attribute do not load when the page is served as XHTML (<code>application/xhtml+xml</code>). See <a href='https://crbug.com/717643'>bug 717643</a>."
                 },
                 "firefox": [
```

Comment:
Upon testing: Firefox data seems fine

Reply:
Interesting, EdgeHTML supported this, I'm surprised! Updating as appropriate.
As for Firefox, it's good to see that they haven't regressed since I last checked.